src/overlays/actors/ovl_En_Hit_Tag/z_en_hit_tag.c

```diff
@@ -29,21 +28,66 @@ const ActorInit En_Hit_Tag_InitVars = {
     (ActorFunc)NULL,
 };
 
-// static ColliderCylinderInit sCylinderInit = {
-static ColliderCylinderInit D_80BE21F0 = {
-    { COLTYPE_NONE, AT_NONE, AC_ON | AC_TYPE_PLAYER, OC1_ON | OC1_TYPE_PLAYER, OC2_TYPE_1, COLSHAPE_CYLINDER, },
-    { ELEMTYPE_UNK0, { 0x00000000, 0x00, 0x00 }, { 0xF7CFFFFF, 0x00, 0x00 }, TOUCH_NONE | TOUCH_SFX_NORMAL, BUMP_ON, OCELEM_NONE, },
+static ColliderCylinderInit sCylinderInit = {
+    {
+        COLTYPE_NONE,
+        AT_NONE,
+        AC_ON | AC_TYPE_PLAYER,
+        OC1_ON | OC1_TYPE_PLAYER,
+        OC2_TYPE_1,
+        COLSHAPE_CYLINDER,
+    },
+    {
+        ELEMTYPE_UNK0,
+        { 0x00000000, 0x00, 0x00 },
+        { 0xF7CFFFFF, 0x00, 0x00 },
+        TOUCH_NONE | TOUCH_SFX_NORMAL,
+        BUMP_ON,
+        OCELEM_NONE,
+    },
     { 16, 32, 0, { 0, 0, 0 } },
 };
 
-#endif
+void EnHitTag_Init(Actor* thisx, GlobalContext* globalCtx) {
+    ColliderCylinder* pCylinder;
```

Comment:
```suggestion
    ColliderCylinder* cylinder;
```
I'm not sure what the significance of the p is in front

Reply:
The p is meant to be a prefix for "pointer". Nevertheless I agree with your style choice and have changed it.